app/src/main/kotlin/com/mapzen/erasermap/view/MainActivity.kt

```diff
@@ -182,6 +184,7 @@ public class MainActivity : AppCompatActivity(), MainViewController, RouteCallba
         app?.onActivityResume()
         autoCompleteAdapter?.clear()
         autoCompleteAdapter?.notifyDataSetChanged()
+        invalidateOptionsMenu()
```

Comment:
Sorry - my mistake. The way this diff was displayed I thought this was in `onCreate()` when it in fact is in `onResume()`.